api/concept/search.py

```
@@ -1,32 +1,37 @@
-        [
-            "%" + SUG.REQ.PST[string.split(" ")[0]] + "%"
-        ]
-    ))
-    return SUG.REQ.RES.get()
\ No newline at end of file
+#
+#   HANDLER
+#
+
+# HANDLER -> LOAD
+from website import *
```

Ruff (0.11.9)

`from website import *` used; unable to detect undefined names

(F403)